common/nwg_classes.cc

```diff
@@ -13,6 +13,10 @@
 #include <array>
 #include <iostream>
 
+#ifdef HAVE_GTK_LAYER_SHELL
+#include <gtk-layer-shell/gtk-layer-shell.h>
```

Comment:
Already included via `nwg_classes.h` (and has the same problem with include path).